CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ Currently the versioning policy of this project follows [Semantic Versioning v2.
 
 ## Unreleased - 2020-??-??
 
+## 4.0.0-RC3 - 2020-02-04
+
+This version contains no change, except for the solution for [a deployment problem](https://issues.sonatype.org/browse/MVNCENTRAL-5548).
```

Comment:
Could you please comment on the PR, what the problem is? I can't access this ticket because I have no sonatype account (do not plan to create one too).

Reply:
Yes it is related to gradle/gradle#11308, Gradle 6 creates more hash for better security, however it does not work with Sonatype Nexus. As a workaround, we disable the feature by feature flag.

Reply:
@iloveeclipse can we merge this change?